WordPress/UITestsFoundation/Screens/NotificationsScreen.swift

```diff
@@ -63,6 +83,25 @@ public class NotificationsScreen: ScreenObject {
         return self
     }
 
+    public func replyToComment(withText text: String) -> Self {
+        waitAndTap(replyCommentButton)
+        replyTextView.typeText(text)
+        replyButton.tap()
+
+        return self
+    }
+
+    @discardableResult
+    public func verifyReplySent(file: StaticString = #file, line: UInt = #line) -> Self {
+        XCTAssertTrue(replyIndicatorCell.waitForExistence(timeout: 5), file: file, line: line)
+
+        let expectedReplyIndicatorLabel = NSLocalizedString("You replied to this comment.", comment: "Text to look for")
+        let actualReplyIndicatorLabel = replyIndicatorText.label.trimmingCharacters(in: .whitespaces)
```

Comment:
for some reason, there's a whitespace on the returned label so had to trim the whitespace to match the expected text